config_operator/config_operator/k8s/K8sConfigMap.py

```diff
@@ -0,0 +1,97 @@
+import os
+import logging
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+
+from config_operator.config_source.exceptions import UnreadableFileException
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class K8sConfigMap:
+    def __init__(self, configmap_name, namespace, git_remote_config):
```

Comment:
`git_remote_config` should be renamed to something else because it could be either a `RemoteGitConfig` or a `LocalDirConfig` object.

Reply:
Yes, I've seen that while answering your comments above. I named it external_config.